week-2/C-comparison-operators/exercise.js

```diff
@@ -8,14 +8,21 @@
 var studentCount = 16;
 var mentorCount = 9;
 var moreStudentsThanMentors; // finish this statement
+if (studentCount > 10) {
+  moreStudentsThanMentors = true;
+}
 
 var roomMaxCapacity = 25;
 var enoughSpaceInRoom; // finish this statement
-
+if (roomMaxCapacity <= 25) {
```

Comment:
Here we are checking if there is enough room for all students and mentors (see the console.log below for direction). So you should be comparing the `roomMaxCapacity` to something else...

Reply:
Thanks, @LucyMac  will be pushing my corrected update tonight.